app/assets/javascripts/components/user_profiles/course_details.jsx

```diff
@@ -2,10 +2,11 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 const CourseDetails = ({ courses }) => {
+  const labelKey = window.Features.wiki_ed ? 'courses.view_page' : 'courses_generic.view_page';
```

Comment:
`window` shouldn't be necessary here, as we normally call `Features.wikiEd` anywhere in the JS that needs to switch behavior based on this setting.

Reply:
Note that we have a helper function that is built to do this kind of switching: `CourseUtils.i18n`. It might be that the necessary data is not directly available, but you should check whether the course_string_prefix is part of the JS state already, which will let you use that utility function instead of implementing behavior switch here.

Reply:
I have removed that. Thank you for helping me understand this!